docs/units/constants_versions.rst

```diff
@@ -11,15 +11,15 @@ Setting the prior versions at the start of a Python session will allow
 consistent units, as follows:
 
 >>> import astropy
->>> astropy.physical_constants.set('codata2010')
+>>> astropy.physical_constants.set('codata2010')  # doctest: +SKIP
```

Comment:
why skipping the doctesting?

Reply:
It raises RuntimeError, because units or constants have already been imported.

Reply:
I think it would still be preferable to show a working example, let me dig up how to reset the pytest namespace.

Reply:
If one can reset the namespace, that might solve the testing problem more generally too...

Reply:
now this is embarrassing as I remembered to this to be something easy to do, but then now can't find how to do it. So I would suggest that we open a separate issue for this for 4.0 to make sure this will be attended with a solution by the time this is released to users.

Reply:
Sounds good. @stargaser - could you raise that issue?

Reply:
I'll open an issue in a moment.